### PR TITLE
feat: add changeRegistry() & add official account logic

### PR DESCRIPTION
**Pull request by gummy789j**

### CODE REVIEW

File: contracts/StrategyProviderWallet.sol [hunk 1]
```diff
@@ -98,7 +105,11 @@ contract StrategyProviderWallet is IStrategyProviderWallet {
         address pool = IUniswapV3Factory(uniswapAddressHolder.uniswapV3FactoryAddress()).getPool(token0, token1, fee);
         require(pool != address(0), "SPWAP0");
 
-        checkPerformanceFeeRatio(performanceFeeRatio);
+        ///@dev check performanceFeeRatio only if the official account is not the owner
+        if (owner != registry.officialAccount()) {
+            checkPerformanceFeeRatio(performanceFeeRatio);
+        }
```
Comment (kyle8615) [inline contracts/StrategyProviderWallet.sol]: add protection for official account config error?

Comment (gummy789j) [reply contracts/StrategyProviderWallet.sol]: what kind of config error?

Comment (kyle8615) [reply contracts/StrategyProviderWallet.sol]: such as rate greater than 100%. if exceeds 100% TX still passes, but when the position fails at closing

**Comment by gummy789j** [comment]
transfer to new PR #5

**Pull request merged**